number_guesser.py

```diff
@@ -0,0 +1,33 @@
+import random
+
+
+def main():
+    number_guess()
+
+
+def number_guess():
+    """a function to generate a random number and has the user guess"""
+    test = 0
+    user_num = int(input("Guess a whole number(Non negative) from 1 to 100: \
+                            "))
+    randnum = random.randint(1, 100)
+    while test == 0:
+        if randnum == user_num:
+            print("You're correct!")
+            play_again = input("play again?")
+            if play_again.upper() == "NO":
+                print("Have a great day!")
+                test = 1
+            else:
+                user_num = int(input("Guess a whole number(Non negative) from \
```

Comment:
might also want to consider actually resetting the value of `randnum` here or call `number_guess()` again so you get a new number to guess.